examples/run-tests.job

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --tasks-per-node=40
+#SBATCH -J hd3tests
+#SBATCH -p short
+#SBATCH -t 0-2:00
+
+# make sure anaconda is activated
+source $HOME/miniconda3/bin/activate
+conda activate haddock3
+
+# go to the example directory
+cd $HOME/haddock3/examples
+
+# execute
+python run_tests.py -b
```

Comment:
I'd change this to be a template instead, this is very particular to our cluster - other's will not have a `short` queue. Best to avoid people just blindly trying to run things.

Reply:
Yeah. Okay. I wrote in the docs the developer should edit it. But I can make it a template. But what parts should I transform the template? Should I remove the `queue` line?

Reply:
Is this is just for internal usage I'd actually remove it. People running HPC jobs are advanced users and can write simple scripts

Reply:
Yes. It is as stated in the documentation. But, yes. Maybe is better to remove that mention and the `run-test.job`.